rewards/rewards_checker.py

```diff
@@ -75,24 +76,21 @@ def verify_rewards(past_tree, new_tree, tree_manager: TreeManager, chain: str):
             chain,
         )
         raise e
-
+    table = []
     for name, token in TOKENS_TO_CHECK[chain].items():
         total_before_token = int(past_tree["tokenTotals"].get(token, 0))
         total_after_token = int(new_tree["tokenTotals"].get(token, 0))
+        decimals = 18
         console.log(name, total_before_token, total_after_token)
         if name == "Digg":
-            diff, table = token_diff_table(
-                name,
-                digg_utils.shares_to_fragments(total_before_token),
-                digg_utils.shares_to_fragments(total_after_token),
-                decimals=9,
-            )
-            print(
-                digg_utils.shares_to_fragments(total_before_token),
-                digg_utils.shares_to_fragments(total_after_token),
-            )
-        else:
-            diff, table = token_diff_table(name, total_before_token, total_after_token)
-        send_code_block_to_discord(
-            msg=table, username="Rewards Bot", url=get_discord_url(chain, BotType.Cycle)
+            total_before_token = digg_utils.shares_to_fragments(total_before_token)
+            total_after_token = digg_utils.shares_to_fragments(total_after_token)
+            decimals = 9
```

Comment:
Not necessary to set decimals here if we use the above comment

Reply:
Fixed